### DISCUSSION TITLE
WirGuard VPN Cluster Access Peer Configuration

### DISCUSSION THREAD

**Discussion by prince-taker** [chpc-tech-eval/scc#155] [Q&A] [answered]
I'm trying to do the following:
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/69721467/5dd735de-7577-41f5-8da5-7f3f66b448e3)
But getting an error message related to PEER_B_PUBLIC_KEY

The error:
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/69721467/58e948af-8a75-420c-919a-12383d933277)

**Comment by mmhashatsi** [comment]
Please read the comment above the command
sudo wg set wg0 peer  instruction.... That ip address is for City Lodge, edit the command with your endpoint

**Reply by prince-taker** [reply]
Hi, I edited the command but still getting the same error:
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/69721467/4fde0477-014b-4f04-ab79-f5c81e307e74)

**Reply by prince-taker** [reply]
Just to confirm we are using the IP adrdress we get from what is my ip website right?

**Reply by nyameko** [reply, marked as answer]
Hi Prince-Taker,

`<PEER_B_PUBLIC_KEY>` is the public wireguard key, from another device, e.g. `laptop.pub`

You will need to create `laptop.pub` on your laptop, the same way you created `headnode.pub`
